src/GovUk.Education.ExploreEducationStatistics.Admin/Settings/PublicDataApiOptions.cs

```diff
@@ -0,0 +1,9 @@
+#nullable enable
+namespace GovUk.Education.ExploreEducationStatistics.Admin.Settings;
+
+public class PublicDataApiOptions
+{
+    public static readonly string Section = "PublicDataApi";
```

Comment:
Minor one, this could be `public const string` rather than `public static readonly string` , but I see it wouldn't be consistent with `PublicDataProcessorOptions` then. Might be best to update both if you decide to change this. 🙂